packages/ketcher-react/src/script/editor/utils/selectItems/isDraggingStructureOnSaltOrSolvent.ts

```diff
@@ -0,0 +1,21 @@
+import { ReSGroup, SGroup } from 'ketcher-core'
+
+export function isDraggingStructureOnSaltOrSolvent(
```

Comment:
isDraggingStructureOnSaltOrSolvent function from the select.ts moved to a dedicated file